### PR TITLE
Changed title to Firefox Extension Workshop

### LINKED ISSUE #592: Change site name to Firefox Extension Workshop
To help improve SEO, we're renaming this site to Firefox Extension Workshop. 

For this issue, we need to update: 

- Header bar: 
Screen Shot 2020-03-17 at 9 14 03 AM (be sure to check how this formats) 

- The site title in `config.yml`

### PR DESCRIPTION
**Pull request by obviyus**
Fixes #592

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # you will see them accessed via {{ site.title }}, {{ site.url }}, and so on.
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: Get help creating & publishing Firefox extensions
```
Comment (championshuttler) [inline _config.yml]: I dont think this is the required changes

Comment (obviyus) [reply _config.yml]: Should I change the description instead?

Comment (muffinresearch) [reply _config.yml]: > Should I change the description instead?

@caitmuenster  "Get help creating & publishing Firefox extensions" probably makes more sense as a description and "Firefox Extension Workshop" as the title (note this changed very recently in https://github.com/mozilla/extension-workshop/commit/d69c58dca42cb739667906ce6bea2a02e187156d) see the difference in titles  between -dev and production, I wonder if that change should be reversed based on the other upcoming changes to titles?

Comment (caitmuenster) [reply _config.yml]: @muffinresearch Can you show me where the `description` property gets surfaced? `d69c58d`looked good based on Raphael's SEO recommendations but I'm ok with reversing it if it makes more sense to move "Get help creating & publishing Firefox extensions" to the desc and changing the title to "Firefox Extension Workshop."

Comment (muffinresearch) [reply _config.yml]: This is the -dev homepage:

```
<title>Get help creating &amp; publishing Firefox extensions | Extension Workshop</title>
<meta name="description" content="Extension Workshop" />
```

Vs Production homepage:

```html
<title>Extension Workshop | Firefox Extension Workshop. Make the web do what you want. If you can dream it, our open souce API and your HTML, CSS, and JavaScript skills can make it happen.</title>
<meta name="description" content="Firefox Extension Workshop. Make the web do what you want. If you can dream it, our open souce API and your HTML, CSS, and JavaScript skills can make it happen." />
```

For the`/documentation/develop/` on -dev:

```html
<title>Develop | Get help creating &amp; publishing Firefox extensions</title>
<meta name="description" content="Extension Workshop" />
```

For the same on prod:

```html
<title>Develop | Extension Workshop</title>
<meta name="description" content="Firefox Extension Workshop. Make the web do what you want. If you can dream it, our open souce API and your HTML, CSS, and JavaScript skills can make it happen." />
```

What we need to think about is what the end state is and if the way we're building titles makes sense.

Right now, there's this global data which is the bit we're talking about in this patch:

https://github.com/mozilla/extension-workshop/blob/b86cdcc494b8fbbfbd119173c8a0953482a1d2f3/_config.yml#L16-L18

We're currently using: https://github.com/jekyll/jekyll-seo-tag/ this uses the title data for each page (from: https://github.com/mozilla/extension-workshop/blob/b86cdcc494b8fbbfbd119173c8a0953482a1d2f3/_data/pages.yaml) and appends the `site.title` the `site.title` comes from the _config.yml mentioned above.

_**Note: That if we update to 11ty this plugin won't exist so we should consider what we want and make sure we can build that out correctly.**_

Comment (caitmuenster) [reply _config.yml]: Thanks for clearing that up, @muffinresearch! Ok, yes, it makes more to set "Firefox Extension Workshop" as the title (even though that changes `d69c58d` and set the description as "Get help creating & publishing Firefox extensions." @obviyus if you want to make that update we can move this patch forward. :)

**Comment by caitmuenster** [comment]
Thank you @obviyus! 🙌🏼 

Having looked at this locally, I now think that we don't need to add the word "Firefox" to the words "Extension Workshop" on the menu bar -- the Firefox logo on the bar is probably sufficient in conjunction with the title change to "Firefox Extension Workshop."

Screen Shot 2020-03-20 at 11 36 33 AM

@muffinresearch, if it's ok with you, I can merge @obviyus' patch in and then back out the change to the title bar. (@obviyus, you're welcome to update the commit yourself but I don't want to ask you to do extra work when you've already been very patient with some back and forth on this patch. 😉 )

**Comment by obviyus** [comment]
@caitmuenster not a problem at all. It's a simple enough change.

**Pull request merged**